unstructured/ingest/connector/notion/connector.py

```diff
@@ -24,6 +27,13 @@ class SimpleNotionConfig(BaseConnectorConfig):
     database_ids: t.List[str]
     recursive: bool
     api_key: str
+    max_retries: t.Optional[int] = None
+    max_time: t.Optional[float] = None
+
+    def get_retry_strategy(self):
+        if self.max_time or self.max_retries:
+            return RetryStrategy(max_time=self.max_time, max_tries=self.max_retries)
+        return None
```

Comment:
are other connectors going to be defining _different_ retry strategies? or mostly re-defining this every time?

Reply:
Figured it made sense to start off with this until we look into how the other connectors might use this, and refactor then.

Reply:
cool, yep, makes sense!